### PR TITLE
Add recordingpage

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: src/components/RecordButton.js [hunk 1]
```diff
@@ -6,15 +6,17 @@ import isRecordingButton from '../images/recording-wave.gif';
...
+  devicesState,
+  setDevicesState,
 }) {
   const [isRecording, setIsRecording] = useState(false);
 
   useEffect(() => {
-    if (recordingSrc !== '') {
+    if (devicesState !== '') {
       setIsRecording(false);
+      setDevicesState('');
```
Comment (mwarnck) [inline src/components/RecordButton.js:17]: Is it the same like in the DrumLoopPlayer.js? Is it needed in both components?

File: src/pages/RecordingsPage.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    <>
+      <HeadingContainer>
+        <BackButton to="/">
+          <img src={backButton} alt="back-button" width="35px" height="35px" />
+        </BackButton>
+        <Heading>My Recordings</Heading>
+      </HeadingContainer>
+      <RecordingsContainer>
+        {myRecordings.map(recording => (
+          <div key={nanoid()}>
```
Comment (mwarnck) [inline src/pages/RecordingsPage.js:19]: Does the nanoid function do not give the recording a new id every time you map over the array?? I am not sure if this is what you want, but maybe I am wrong or this doesn´t matter for your function :)

**Review by mwarnck — APPROVED** [review]
Looks overall very good to me. Have a look at my comments.

File: src/components/RecordButton.js (hunk 1 above)
Comment (FaGru) [reply src/components/RecordButton.js:17]: Yes, the audio recorder and the drum loop player have to stop each other in certain cases. This state then changes the image of the buttons to the initial value.

File: src/pages/RecordingsPage.js (hunk 2 above)
Comment (FaGru) [reply src/pages/RecordingsPage.js:19]: Thanks you are right, maybe it is better on a other way, but in this case it doesn't matter 😅

**Pull request merged**